src/borg/helpers/parseformat.py

```diff
@@ -834,13 +837,31 @@ def calculate_csize(self, item):
         # note: does not support hardlink slaves, they will be csize 0
         return item.get_size(compressed=True)
 
-    def hash_item(self, hash_function, item):
-        if 'chunks' not in item:
-            return ""
+    def prepare_hash_function(self, hash_function):
```

Comment:
This does not "prepare" the hash function in any way, it just picks one by its name, which is what one passes into it, rather than a function, thus naming this something like `get_hash_function(self, name)` would be less misleading and more self-explanatory.